packages/terra-tabs/src/common-tabs/CommonTabContent.module.scss

```diff
@@ -8,16 +8,22 @@
     flex-direction: column;
     height: 100%;
     overflow: hidden;
-    padding: 0 10px;
     position: relative;
     width: 100%;
   }
 
+  .panel-padding {
+    padding: 0 10px;
+  }
+
   .panel-header {
     flex: 0 0 auto;
+    z-index: 10;
```

Comment:
curious, how did we arrive at z-index 10..??

Reply:
this is an existing content style (from workspace) seen in https://github.com/cerner/terra-framework/pull/1807/files#diff-133172bdc7e9047a876b59a63ac335d93cdefd591f1412bfd70c64d716fe722aL20, the change made was to move padding under panel-heading-padding retaining z-index as it is under panel-header.